plugins/videobasedtracker/IdentifierHelpers.h

```diff
@@ -62,17 +62,18 @@ namespace vbtracker {
     /// @brief Helper for implementations of LedIdentifier to turn a
     /// brightness list into a boolean list based on thresholding on the
     /// halfway point between minimum and maximum brightness.
-    inline LedPattern getBitsUsingThreshold(const BrightnessList &brightnesses,
+    inline LedPatternWrapped getBitsUsingThreshold(const BrightnessList &brightnesses,
                                             float threshold) {
-        LedPattern ret;
+        LedPatternWrapped ret;
         // Allocate output space for our transform.
         ret.resize(brightnesses.size());
 
-        // Transform the brightnesses into a container of bools using this
-        // little lambda
+        // Transform the brightnesses into a string with '.' for dim
+        // and '*' for bright.
         std::transform(
             begin(brightnesses), end(brightnesses), begin(ret),
-            [threshold](Brightness val) { return val >= threshold; });
+            [threshold](Brightness val) {
+          if (val >= threshold) { return '*'; } else { return '.'; } });
```

Comment:
can clang format this. Also, might be actually clearer with the ternary comparison operator: `return val >= threshold? '*' : '.';`